Slim/Plugin/Podcast/Plugin.pm

```diff
@@ -328,14 +321,18 @@ sub searchHandler {
 }
 
 sub registerProvider {
-	my ($class, $name, $provider, $force) = @_;
+	my ($class, $force) = @_;
+
+	eval "require $class";
+	my $name = $class->getName;
```

Comment:
You should check `$@` and return quickly, as `$class->getName` would fail if the `require` failed.

Reply:
agreed, although I did not pay attention because I can't see how the dev could miss that.